src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceSpecCallbackHandler.java

```diff
@@ -298,11 +297,12 @@ private CallbackResponse populateCaseData(CallbackParams callbackParams) {
             updatedCaseData.showResponseOneVOneFlag(setUpOneVOneFlow(caseData));
             updatedCaseData.respondent1PaymentDateToStringSpec(setUpPayDateToString(caseData));
 
-            BigDecimal howMuchWasPaid = Optional.ofNullable(caseData.getRespondToAdmittedClaim())
-                .map(RespondToClaim::getHowMuchWasPaid).orElse(null);
-            if (howMuchWasPaid != null) {
-                updatedCaseData.partAdmitPaidValuePounds(MonetaryConversions.penniesToPounds(howMuchWasPaid));
-            }
+            Optional<BigDecimal> howMuchWasPaid = Optional.ofNullable(caseData.getRespondToAdmittedClaim())
+                .map(RespondToClaim::getHowMuchWasPaid);
+
+            howMuchWasPaid.ifPresent(howMuchWasPaidValue ->
+                                         updatedCaseData.partAdmitPaidValuePounds(
+                                             MonetaryConversions.penniesToPounds(howMuchWasPaidValue)));
```

Comment:
Please, could you format this file, there are few lines without format.

Reply:
Updated the file